<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_calc</class>
 <widget class="QDialog" name="credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_different">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>61</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_term">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>45</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>36</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_overpayment">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid black;
	background-color: rgba(64, 65, 66, 242);
	color: white;
}</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>145</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>5</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>5</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_bid">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>45</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип платежа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>115</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата по кредиту:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border:  1px solid black;
	background-color: orange;
	color: black;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop: 1 rgb(255, 138, 100));
}</string>
   </property>
   <property name="text">
    <string>Расссчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>122</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid black;
	background-color: rgba(64, 65, 66, 242);
	color: white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>5</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок, мес</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_sum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>45</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: white;
}</string>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="value">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_annuity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>109</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
